react-backend/database/queries/users_filtered.js

```diff
@@ -55,7 +55,7 @@ const buildAgeQuery = ageQuery =>
           const [low, high] = ageRange.split('-').map(num => Number(num));
           const edgeCaseQuery = {
             '0-17': `(users.yearofbirth > ${getDate(high)})`,
-            '70-more': `(users.yearofbirth <= ${getDate(low)})`,
+            '70-more': `(users.yearofbirth <= ${getDate(low)})`
```

Comment:
Is this prettier removing all these dangling commas? 

Can you add `"comma-dangle": ["error", "always-multiline"]` to the root-level `.eslintrc.json`? Then these can be auto-fixed.

Reply:
There should be a prettier config setting to keep them also `"prettier.trailingComma": "es5"`, I think.